examples/simple/igraph_assortativity.c

```diff
@@ -0,0 +1,30 @@
+#include <igraph.h>
+#include <stdio.h>
+
+int main(){
+    igraph_t g;
+    igraph_vector_t types;
+    igraph_real_t assortativity;
+
+    /* Create directed graph */
+    igraph_small(&g, 10, IGRAPH_DIRECTED, 
+                 0, 1, 0, 2, 0, 3, 0, 5,
+                 3, 6,
+                 4, 5,
+                 6, 4, 6, 7,
+                 7, 8, 7, 9,
+                 8, 9,
+                 9, 0,
+                 -1);
+
+    igraph_vector_init(&types, 0);
+    igraph_degree(&g, &types, igraph_vss_all(), IGRAPH_ALL, /* Consider self-loops */ IGRAPH_LOOPS);
```

Comment:
tiny nitpick, just for consistency

```suggestion
    igraph_degree(&g, &types, igraph_vss_all(), IGRAPH_ALL, /* consider self-loops */ IGRAPH_LOOPS);
```

If you like, you can click the "merge" button here, then `git pull` on your side